hack/install-ocs-operator.sh

```diff
@@ -39,7 +39,6 @@ EOF
 
 oc wait --timeout=5m --for condition=Available -n "$INSTALL_NAMESPACE" deployment \
     ceph-csi-controller-manager \
-    csi-addons-controller-manager \
```

Comment:
This change should be part of the first commit.

Reply:
I can change but don't want to re-trigger whole OCP and OCS deploy, could you override that?

Reply:
I can do that not an issue.

Reply:
done.